openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/Task.spec.js

```diff
@@ -52,6 +52,7 @@ describe('Task flow should work', () => {
     verifyResponseStatusCode('@suggestApi', 200);
 
     cy.get(`[data-testid="assignee-option-${secondAssignee}"]`)
+      .should('be.visible')
```

Comment:
This is to avoid flakiness.

Reply:
Cypress advice to fire event on freshly fetched elements instead of an older reference to avoid flakiness
You can assert that should be visible and then fetch it again to perform action on the same element